### DISCUSSION TITLE
need documents for config file parameters

### DISCUSSION THREAD

**Discussion by parichehrvn** [open-edge-platform/anomalib#923] [Q&A]
Hi, 
The usage of some parameters are not clear. It is better to prepare a documentation for config files. For example could you please describe the following parameters? what are they applications?
**deterministic, detect_anomaly, strategy, adaptive threshold**
Thanks.

**Comment by samet-akcay** [comment]
Hi @parichehrvn, those are  PyTorch Lightning's `Trainer` parameters. PL already has a great documentation. We could perhaps update our documentation by referring to theirs.
https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html

**Comment by parichehrvn** [comment]
thank you. The link was really helpful.
